CMakeLists.txt

```diff
@@ -149,41 +158,44 @@ find_library(LIB_GPR NAMES gpr)
 target_link_libraries(${EXE} PRIVATE ${LIB_GRPC_UNSECURE} ${LIB_GRPC} ${LIB_GPR})
 
 # Find Protobuf
-find_library(LIB_PROTOBUF NAMES libprotobufd)
-find_library(LIB_PROTOC NAMES libprotocd)
-find_library(LIB_PROTOLITE NAMES libprotobuf-lited)
-target_link_libraries(${EXE} PRIVATE ${LIB_PROTOC} ${LIB_PROTOBUF} ${LIB_PROTOLITE})
+include(FindProtobuf)
+include_directories(${Protobuf_INCLUDE_DIRS})
+target_link_libraries(${EXE} PRIVATE ${Protobuf_LIBRARIES})
```

Comment:
Same comment here too, why do this for protobuf but not grpc.

Reply:
I already answered this over on the enigma-dev PR, there is a `FindGRPC` that was added recently but it's not well documented.